CSA.md

```diff
@@ -1,91 +1,93 @@
 # Cloud Service Agreement
 
 ## USING THIS AGREEMENT
-This Agreement allows <span class="coverpage_link">Customer</span> to access and use the Cloud Service. To use this Agreement, the parties must complete and sign or electronically accept a Cover Page and Order Form.
-Capitalized and highlighted words have the meanings given on the Cover Page or in the Order Form. However, if the Cover Page or Order Form omit or do not define a highlighted word, the default meaning will be "none" or "not applicable". All other capitalized words have the meanings given in these Standard Terms.
+To use this Agreement, the parties must complete and sign or electronically accept a Cover Page.
+Capitalized and <span class="coverpage_link">highlighted</span> words have the meanings given on the Cover Page. However, if the Cover Page omits or do not define a highlighted word, the default meaning will be "none" or "not applicable". All other capitalized words have the meanings given in these Standard Terms.
 
 1. ## Service  
 
-    1.1 <span class="header_3">Access and Use.</span>  During the <span class="orderform_link">Subscription Period</span> and subject to the <span class="orderform_link">Use Limitations</span>, <span class="coverpage_link">Customer</span> may access and use the Cloud Service for its internal business purposes. If the Cloud Service includes Software or Documentation, <span class="coverpage_link">Customer</span> may copy and use the Software and Documentation only as needed to access and use the Cloud Service and only if <span class="coverpage_link">Customer</span> complies with the terms of this Agreement.
+    1.1 <span class="header_3">Access and Use.</span>  During the <span class="orderform_link">Subscription Period</span> and subject to the <span class="orderform_link">Use Limitations</span>, <span class="coverpage_link">Customer</span> (a) may access and use the Cloud Service and (b) copy and use the included Software and Documentation only as needed to access and use the Cloud Service, in each case, for its internal business purposes and only if <span class="coverpage_link">Customer</span> complies with the terms of this Agreement.
```

Comment:
Global comment to update the coverpage_link span class to keyterms_link span class. The Cover Page is the umbrella term for the customizable part of our agreements, and includes both the Order Form and the Key Terms. All terms classed to Order Form should remain the same, but I'll double check those, too.